src/components/comment-placer/CommentPlacer.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { findDOMNode } from 'react-dom';
+import FocusManager from './focus-manager';
+import FadeAndGrowTransition from './fade-and-grow-transition';
+import FadeTransition from './fade-transition';
+
+export default class CommentPlacer extends Component {
+    static propTypes = {
+        animateOnMount: PropTypes.bool,
+        animateOnUnmount: PropTypes.bool,
+        scrollOnMount: PropTypes.bool,
+        autofocus: PropTypes.bool,
+        children: PropTypes.element.isRequired,
+        animation: PropTypes.oneOf(['fade', 'fade-and-grow']).isRequired,
+        className: PropTypes.string,
+        listHasScroll: PropTypes.bool.isRequired,
```

Comment:
Why is this property needed?

Reply:
Somehow, when there's no scroll on the list (I mean when there are not enough comments to have scroll on the list), when we make a new comment, the intersection observer always returns `inView: false`. However, the intersection observer is working properly (or as expected) when there is scroll on the list and thus, this is not happening.

Reply:
Strange. I will try to replicate this locally and see why.

Reply:
Thanks!

Reply:
@PedroMiguelSS I couldn't reproduce the issue. I did `console.log(isIntersecting)` in the `handleObserverChange` and it is true, even if the list has only 1 comment and has no scroll.

Reply:
Did you try to reproduce it only for new comments, right? Using the bottom bar.

Reply:
Right

Reply:
Will check it again 🤔

Reply:
You're right. Seems it's working properly. I'll remove that `prop`.